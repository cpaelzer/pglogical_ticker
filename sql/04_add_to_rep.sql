SELECT pglogical_ticker.add_ticker_tables_to_replication();
