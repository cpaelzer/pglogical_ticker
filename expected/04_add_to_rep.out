SELECT pglogical_ticker.add_ticker_tables_to_replication();
 add_ticker_tables_to_replication 
----------------------------------
                               11
(1 row)

